Week04/lesson08/homework/temp_converter/js/indexbonus4.js

```diff
@@ -0,0 +1,153 @@
+function conversion() {
+
+  var c = document.querySelector("#c").value;
+  var f = document.querySelector("#f").value;
+
+  if ((f === null || f === "") && (c === null || c === "")) {
```

Comment:
Since you're checking for both `f === null` and `f === ""` (and ditto with `c`), I'd recommend isolating these in a variable:

```
var fNotFilledIn = f === null || f === "";
var cNotFilledIn = c === null || c === "";

if (fNotFilledIn && cNotFilledIn) {
  // do stuff
} else if (fNotFilledIn) {
  // do stuff
} else {
  // do stuff
}
```